packages/hooks/src/test/index.test.js

```diff
@@ -0,0 +1,4 @@
+/**
+ * Internal Dependencies
+ */
+import { doAction, applyFilters, addAction, addFilter, doingAction, doingFilter, didAction, didFilter, hasAction, hasFilter, removeAllActions, removeAllFilters, currentFilter } from '../';
```

Comment:
We could also `export default { doAction, ... }` and then just `import hooks from '../`, would likely be a bit nicer.